### ISSUE TITLE
You Tube

### ISSUE THREAD

**Issue by cgroveacademy** [OpenLiveWriter/OpenLiveWriter#423]
I'm trying to embed a youtube video to my post and it has freezed up the entire app and now keeps shutting down.  I can't even delete the code to fix it, bc it shuts down as soon as I open that post now.

**Comment by cgroveacademy** [comment]
Okay, I was able to get in and delete the code, but I still need to post the youtube video, it's a review video that needs to go with the post and it's due tomorrow :/  Thanks!

**Comment by ScottIsAFool** [comment]
What blog type is this for?

**Comment by cgroveacademy** [comment]
sorry It's blogger, but it's freezing up in OLW.  I did it before, just cut and pasted the embed code from youtube for another post a while back and it was fine.

**Comment by Laronski** [comment]
I personally haven't had any problems with inserting/embedding YouTube videos with Open Live Writer.
 
I think it's important to note that Open Live Writer uses Internet Explorer/Microsoft Edge HTML engine (for now) for its renderings; so to be on safe side, ensure you have the latest system patches and the latest version of Internet Explorer/Microsoft Edge.

Just trying to help...

**Comment by cgroveacademy** [comment]
I have not had issues in the past either.  It's weird.  Everything is up to date.  It just won't let me add anything more to it after I put the youtube video, I just uploaded it as a draft to blogger and am going to try finishing the post that way for now.

**Comment by ScottIsAFool** [comment]
Just to check, what happens if you turn the blog theme off in live writer first, before opening the post?

**Comment by cgroveacademy** [comment]
still doesn't work

**Comment by cgroveacademy** [comment]
I was able to complete the post in Blogger... but I hate to do that :/  lol

**Comment by elmer0224** [comment]
After I sign-in to YouTube (inside Open Live Writer), it gives me the error, having problems getting the videos". I'm getting it from my Favorites list on YouTube and I have no problem logging in. Please help :(

![screenshot 2016-03-23 10 08 46](https://cloud.githubusercontent.com/assets/15074275/13973611/7f52187c-f0df-11e5-90f5-f759583ad8ad.png)

**Comment by Laronski** [comment]
For me, that function hasn't worked for a really long time. I think there's going to be a new plug-in for Open Live Writer to insert videos. 

In regards to inserting videos, I'd like to see Open Live Writer be able to understand pasted YouTube, and other video services, and simply insert the videos.

**Comment by ScottIsAFool** [comment]
The Youtube stuff needs moving to newer Google APIs.

**Comment by nbridges** [comment]
I have Open Live Writer, running it on windows 7 starter.
On install it was not able to download theme. 
I ignored this but am not sure if this was the correct thing to do, because now I realize I also have other issues. 
Opinions & Advice please....

I have posted from OLW to googles blogger without issue.

Today I have tried to log into youtube through OLW and recieved an error message telling me it does not recognize google. I thought maybe I should use the begining of my email address for logging in (like you can do on google when signing into email)  Leaving off the @gmail.com part. This too did nothing and I was unable to log into the youtube on OLW. 
I wanted to insert a youtube video along with my text on blogger, without visiting youtube & blogger to ensure the video was part of the post. 

I also tried to use the "From Web" part of the add video in OLW and it also failed to work for me.

For me, so far, It has been a complete fail with attempting to use the youtube sign in and add a video.

At this moment these are the only issues I have encountered. 

Are there any patches, plug in's or anything fixes for these issues?

Has anyone else had these issues?

Any advice on how to log into youtube through OLW?

**Comment by itzanopinion** [comment]
Been having a similar issue as well trying to log into YT using OLW!

Once I click on "insert video" and sign into Youtube, OLW comes back with the following message:

**_"There was an error getting the list of videos!"_**

No other option of trying to insert any of the few videos I have under my YT page works, such as selecting "From Web" and using the URL...that results in a different error message, which states:

**_"Unknown Provider or incomplete web address."_**

Sure would like to get this resolved, as having to bounce back-and-forth between the actual blogger GUI and YT is a P.I.T.A.

FWIW...I'm trying to post to a blogspot blog.

**Comment by kgburgess** [comment]
Ditto on everything everybody said.  I write curriculum reviews and love to use embedded videos to personalize the reviews.  I have been unable to do so for all of the same reasons.  Very frustrating.  Please fix!

**Comment by sboyd000** [comment]
Has anyone been able to get a youtube video linked in Open Live Writer?  I've tried the video icon where when I sign into youtube with the correct login and password that works in youtube OLW says Blogger returned the following error: not found, not found.    If I try to insert the link youtube gave me to the videos I published then anyone other than me gets the link cut off and an error that says videos not found.   What am I doing wrong here??

**Comment by kgburgess** [comment]
Yes, sboyd000...I found that if I just add the embed code for the YouTube video on the HTML tab, then a black box appears and says there's an error.  But when I actually publish the post with that in it, the actual video does publish on my blog.  So while you can't preview it in your post as you write it, it does actually work when you publish to your blog.  I hope that helps.

**Comment by shanselman** [comment]
Thanks for reporting this and for helping Open Live Writer over the years. We are doing an issue-bankruptcy cleanup so we can make the repository maintainable again and focus on current, actionable work. This older issue is being closed as part of that cleanup, not because it was unimportant. If this still reproduces in the current code or is still relevant to the revitalization work, please open a fresh issue with current details and we will triage it going forward.
